kernel/src/process.rs

```diff
@@ -489,11 +489,12 @@ unsafe fn parse_and_validate_tbf_header(address: *const u8) -> Option<TbfHeader>
             // Calculate checksum. The checksum is the XOR of each 4 byte word
             // in the header.
             let mut chunks = tbf_header_base.header_size as usize / 4;
-            let mut leftover_bytes = 0;
-            if chunks * 4 != tbf_header_base.header_size as usize {
+            let mut leftover_bytes = if chunks * 4 != tbf_header_base.header_size as usize {
```

Comment:
I don't think this needs to be `mut` now either

Reply:
Thanks, nice catch Pat :)

Weird that I didn´t get a warning for this.